packages/decorators/find-replace/src/decorateFindReplace.ts

```diff
@@ -10,7 +10,7 @@ export const decorateFindReplace: Decorate<{}, FindReplacePlugin> = (
 
   if (search && Text.isText(node)) {
     const { text } = node;
-    const parts = text.split(search);
+    const parts = text.toLowerCase().split(search.toLowerCase());
```

Comment:
I am fine with the change in default behavior to case insensitive.

I would prefer to see a configuration option for someone to opt-in to the existing case sensitive behavior.